src/main/kotlin/com/exactpro/th2/rptdataprovider/messageProfile/ProfilesStatistics.kt

```diff
@@ -0,0 +1,104 @@
+package com.exactpro.th2.rptdataprovider.messageProfile
+import com.fasterxml.jackson.annotation.JsonRawValue
+import com.google.gson.Gson
+import com.google.gson.JsonObject
+import org.apache.log4j.Logger
+import kotlin.system.measureTimeMillis
+
+class ProfilesStatistics {
+    private val logger = Logger.getLogger("worstLogger")
+    var enough = false
+    // long - processing time in ms
+    private var oneSecondWorstList : MutableMap<Long, MessageProfile> = mutableMapOf()
+    private var timerThread : Thread? = Thread {
+        while (!enough) {
+//            Thread.sleep(300)
+//            printArrayInLogs(MessageProfile.getStatistics())
+            Thread.sleep(1000)
+            calculateStatistics()
+        }
+    }
+
+    companion object{
+        @JsonRawValue
+        var body: String? = ""
+
+        private var bestProfileInfo : String = ""
+        private var worstProfileInfo : String = ""
+        private var middleProfileInfo : String = ""
+
+        fun getStatisticsJson() : JsonObject{
+            val resultJson = JsonObject()
```

Comment:
We definitely need to use 2 separate libraries for serialization in json? Is it possible to make a data class with the same fields and serialize using jacksonMapper.asStringSuspend as it is done for example with Message